### DISCUSSION TITLE
Does uBO blocks Youtube's adblocker-detection, or just adblocker-blocking?

### DISCUSSION THREAD

**Discussion by TiagoTiago** [uBlockOrigin/uAssets#20103] [Q&A] [answered]
Is there a risk Youtube will be adding up "strikes" and just not warning you're about to get your account banned, or when the filters/rules are working right Youtube will not even be able to tell anything is going on? And does it just block the ads functionality, or does it also stops tracking data from being sent out to the ad service?

**Comment by Yuki2718** [comment, marked as answer]
https://www.reddit.com/r/uBlockOrigin/comments/173jmog/comment/k4jzusu/

**Comment by VertofChest** [comment]
Honestly uBlock Origin isn't really that great for this thing. Youtube's detection uses scripts, something that **Tampermonkey** can and will do better than simply blocking/removing a page element ever will do. Despite all attempts I could never get UBo to play nice with Youtube even after clearing cookies.

**Reply by ghost** [reply]
Did you even read the answer linked?

**Reply by stephenhawk8054** [reply]
> simply blocking/removing a page element ever will do

uBO does **NOT** block/remove page element with youtube's anti-adb.

---

Right now if you see anti-adblock, it's because of your custom filters/filter lists or other extensions.

**Reply by gorhill** [reply]
@VertofChest 

> simply blocking/removing a page element ever will do

uBO has its own [internal user scripts](https://github.com/gorhill/uBlock/wiki/Resources-Library), and this is exactly what we use for countless anti-blocker cases out there.

One of the main issues we have to deal with is people out there thinking they know very will how uBO works on the technical side, while they actually do not and give bad advice to others as a result.

**Reply by arejfour** [reply]
I'm only having this problem in Chrome. UBO is blocking the YouTube pop up warning in Edge and Firefox. So I'm confused as to why UBO works in Edge and FF and not Chrome. I did notice that UBO version in FF is 1.53 while in Chrome it's 1.52.  I tried the TamperMonkey extension with no effect. I read some of the suggested solutions in a Reddit post. To be honest it's too complicated for me.

**Reply by MasterKia** [reply]
@arejfour Read and do everything in the first post here:
https://github.com/uBlockOrigin/uAssets/issues/19976

If it didn't work, provide your troubleshooting information there.

**Reply by stephenhawk8054** [reply]
@arejfour Disable all other extensions and test again. The more you use, the easier you get anti-adblock
